web/src/styles.css

```diff
@@ -999,6 +1000,10 @@ body {
     cursor: pointer;
 }
 
+#language-header {
+    display: none;
+}
+
```

Comment:
this is overwritten below, should be removed.

Reply:
It's overwritten because I don't want to show this header for desktops. The reason is that for desktops there is a tooltip on the button opening the language menu.

`display: none;` is in media query.

Reply:
my bad! :)